doc/neps/nep-0046-sponsorship-guidelines.rst

```diff
@@ -0,0 +1,254 @@
+.. _NEP46:
+
+=====================================
+NEP 46 — NumPy Sponsorship Guidelines
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Status: Draft
+:Type: Process
+:Created: 2020-12-27
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This NEP provides guidelines on how the NumPy project will acknowledge
+financial and in-kind support.
+
+
+Motivation and Scope
+--------------------
+
+In the past few years the NumPy project has gotten significant financial
+support, as well as dedicated work time for maintainers to work on NumPy. There
+is a need to acknowledge that support - funders and organizations expect or require
+it, it's helpful when looking for new funding, and it's the right thing to do.
+Furthermore, having a clear policy for how NumPy acknowledges support is
+helpful when searching for new support.
+
+This NEP is aimed at both the NumPy community - who can use it when looking for
+support and acknowledging existing support - and at past, current and
+prospective sponsors, who often want or need to know what they get in return
+for their support other than a healthier NumPy.
+
+The scope of this proposal includes:
+
+- direct financial support, employers providing paid time for NumPy maintainers
+  and regular contributors, and in-kind support such as free hardware resources or
+  services.
+- where and how NumPy acknowledges support (e.g., logo placement on the website).
+- the amount and duration of support which leads to acknowledgement.
+- who in the NumPy project is responsible for sponsorship related topics, and
+  how to contact them.
+
+
+How NumPy will acknowledge support
+----------------------------------
+
+There will be two different ways to acknowledge financial and in-kind support,
+one to recognize significant active support, and another one to recognize
+support received in the past and smaller amounts of support.
+
+Entities who fall under "significant active supporter" we'll call Sponsor.
+The minimum level of support given to NumPy to be considered a Sponsor are:
+
+- $30,000/yr for unrestricted financial contributions
+- $60,000/yr for financial contributions for a particular purpose
+- $100,000/yr for in-kind contributions
```

Comment:
I anticipate that this section of the document will get the most attention. Perhaps, we should specify in parenthesis the most common and/or sought out types of in-kind contributions NumPy accepts, e.g. services, hardware, software...

Reply:
Time of employees to contribute is the main one. And services, but those won't need this policy because they're typically "free for all open source projects" kind of services (e.g. CI services).

Realistically, contributor time is the only one that could add up to $100k. Do you think it would help to add "(e.g., time for employees to contribute)"?

Reply:
I believe so. As a matter of fact, let’s detail each category. Not everyone is well versed in the nuances of nonprofit accounting. 
_The minimum level of support given to NumPy to be considered a Sponsor are:
$30,000/yr for unrestricted financial contributions (e.g., donations, grants)
$60,000/yr for financial contributions for a particular purpose (e.g., grants)
$100,000/yr for in-kind contributions (e.g., time for employees to contribute)_

Reply:
good idea, done